spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/deployer/DefaultReleaseManager.java

```diff
@@ -226,8 +226,12 @@ private void updateCountProperty(Map<String, Object> model, String appsCount) {
 	}
 
 	public Release status(Release release) {
+		if (release.getInfo().getStatus().equals(StatusCode.DELETED)) {
```

Comment:
Btw, why all these changes are needed outside of a tests?

Reply:
This was part of the changes I had for a different PR that got merged: https://github.com/spring-cloud/spring-cloud-skipper/pull/671

Reply:
@jvalkeal Not at all! Rebased the PR. Thank you!